### ISSUE TITLE
Custom mesh and BufferGeometry in threejs

### ISSUE THREAD

**Issue by RyugaRyuzaki** [repalash/threepipe#5]
I try to load Ifc Model, i can create geometry and mesh, how can i load it in viewer?

**Comment by repalash** [comment]
If you have a three.js Mesh object, you can directly add it to the viewer scene with `viewer.scene.addObject(mesh)`. This will add the object to a child in the scene object, that can be accessible with `viewer.scene.modelRoot`

**Comment by RyugaRyuzaki** [comment]
@repalash How about InstanceMesh? and can i create custom culler?

**Comment by repalash** [comment]
Yes any kind of Object3D like InstancedMesh can be added in the same way. You can also have a look at the plugins in the extra-importers package to support a new file type with a threejs loader. 

For custom Culler, what kind are you thinking of?

**Comment by RyugaRyuzaki** [comment]
> Yes any kind of Object3D like InstancedMesh can be added in the same way. You can also have a look at the plugins in the extra-importers package to support a new file type with a threejs loader.
> 
> For custom Culler, what kind are you thinking of?

1. I used [this library](https://github.com/ThatOpen/engine_fragment/blob/main/packages/fragments/package.json). All support for >=@threejs 0.160.1

2. For Culler I use [this](https://github.com/ThatOpen/engine_components/blob/main/packages/front/src/fragments/IfcStreamer/src/geometry-culler-renderer.ts).

3. Looks like your library doesn't have support for different threejs versions
4. More example with [GLTFExporter](https://github.com/mrdoob/three.js/blob/dev/examples/jsm/exporters/GLTFExporter.js) of threejs now support InstanceMesh.
5. I want to use [this](https://github.com/zeux/meshoptimizer). to compress glb 
* I can conclude that your library is tying up a lot of threads even though the quality of this library is very high
